plugins/inputs/netdata/netdata.go

```diff
@@ -0,0 +1,291 @@
+package netdata
+
+import (
+	"bytes"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+const chartsURL = "/api/v1/charts"
+
+const description = `Read metrics from netdata`
+
+const sampleConfig = `
+  ## Number of points to gather.
+  ## By default inputs plugin run every 10 seconds, by asking 10 points
+  ## to netdata we get measurement for each second. This value is capped by "update every" config of netdata.
+  ## For example you can set
+  ## points = 20
+  ## interval = "10m"
+  ## to get measurement every 30 seconds (and polling netdata every 10 minutes)
+  ## default to 10
+  points = 10
+
+  ## Grouping method if multiple collected values are to be grouped in order to return fewer points.
+  ## methods supported "min", "max", "average", "sum", "incremental-sum"
+  ## cf : https://github.com/firehol/netdata/wiki/REST-API-v1
+  ## default to average
+  group = "average"
+
+  ## Should measurement be prefixed by the netdata hostname ?
+  ## Ex : netdata.host1.apps.cpu (with hostname),  netdata.apps.cpu (without hostname)
+  ## default to true
+  prefixChartIDByHostname = true
+
+  ## Netdata servers
+  [[inputs.netdata.servers]]
+    url="http://netdata-host:19999"
+
+  [[inputs.netdata.servers]]
+    url="https://netdata-host2"
+    basicAuthUsername="user"
+    basicAuthPassword="pass"
```

Comment:
Please specify a minimal viable config here i.e. leave the second server commented, same for _points_, _group_ and _prefixChartIDByHostname_  representing their respective default already.

Reply:
Mandatory.